packages/quantic/cypress/e2e/default-1/case-classification/case-classification.cypress.ts

```diff
@@ -889,4 +893,343 @@ describe('quantic-case-classification', () => {
+          Expect.displaySelectTitle(false);
+          Expect.displaySelectInput(false);
+        });
+
+        scope('when fetching suggestions', () => {
+          const firstSuggestionIndex = 0;
+
+          mockCaseClassification(
+            coveoDefaultField,
+            allOptions.slice(0, suggestionsCount)
+          );
+          fetchClassifications();
+          Expect.displaySelectTitle(false);
+          Expect.numberOfSuggestions(suggestionsCount);
+          Expect.numberOfInlineOptions(allOptions.length - suggestionsCount);
+          Expect.displaySelectInput(false);
+          Expect.logClickedSuggestion(firstSuggestionIndex, true);
+          Expect.correctValue(allOptions[firstSuggestionIndex].value);
+        });
+
+        scope('when selecting an inline option', () => {
+          const clickedIndex = 0;
+
+          Actions.clickInlineOption(clickedIndex);
+          Expect.logUpdatedClassificationFromInlineOption(
+            coveoDefaultField,
+            clickedIndex
+          );
+          Expect.correctValue(
+            allOptions[clickedIndex + suggestionsCount].value
+          );
+          Expect.fetchClassificationsAfterValueChange(
+            coveoDefaultField,
+            allOptions.slice(0, suggestionsCount)
+          );
+        });
+      });
+    });
+
+    describe('when an option from the select input is selected', () => {
+      it('should automatically fetch new case classifications', () => {
+        const suggestionsCount = 3;
+        visitCaseClassification({
+          fetchClassificationOnChange: true,
+        });
+
+        scope('when loading the page', () => {
+          Expect.displayLabel(true);
+          Expect.numberOfInlineOptions(0);
+          Expect.numberOfSuggestions(0);
+          Expect.displaySelectTitle(false);
+          Expect.displaySelectInput(true);
+        });
+
+        scope('when fetching suggestions', () => {
+          const firstSuggestionIndex = 0;
+
+          mockCaseClassification(
+            coveoDefaultField,
+            allOptions.slice(0, suggestionsCount)
+          );
+          fetchClassifications();
+          Expect.displaySelectTitle(true);
+          Expect.displaySelectInput(false);
+          Expect.numberOfSuggestions(suggestionsCount);
+          Expect.correctSugestionsOrder(allOptions.slice(0, suggestionsCount));
+          Expect.numberOfInlineOptions(0);
+          Expect.logClickedSuggestion(firstSuggestionIndex, true);
+          Expect.correctValue(allOptions[firstSuggestionIndex].value);
+        });
+
+        scope('when selecting a suggestion', () => {
+          const clickedIndex = 1;
+
+          Actions.clickSuggestion(clickedIndex);
+          Expect.logClickedSuggestion(clickedIndex);
+          Expect.logUpdatedClassificationFromSuggestion(
+            coveoDefaultField,
+            clickedIndex
+          );
+          Expect.correctValue(allOptions[clickedIndex].value);
+        });
```

Comment:
Why do we need this section?
I don't think we need a step to select a suggestion because the goal of the describe section is to click on an option from the select input and confirm that a new classification request has been sent.

Reply:
You are right, I must have forgotten to remove this while refactoring. Good catch!